client/blocks/reader-post-card/index.jsx

```diff
@@ -287,7 +243,7 @@ class ReaderPostCard extends Component {
 		const onClick = ! isPhotoPost && ! compact ? this.handleCardClick : noop;
 		return (
 			<Card className={ classes } onClick={ onClick } tagName="article">
-				{ ! compact && ! isReaderTagPage && postByline }
+				{ ! compact && ! isReaderTagPage }
```

Comment:
if we really are going to remove the postByline for all cases, we could remove this entire line.